modules/@apostrophecms/asset/lib/refresh-on-restart.js

```diff
@@ -7,11 +7,15 @@
   var fast = '1';
   check();
   function check() {
+    if (!window.apos) {
+      // Wait for the js bundle to evaluate
+      return setTimeout(check, 100);
```

Comment:
We are using the "module/nomodule" pattern to avoid the overhead of loading the IE11 build in modern browsers. Since we are using the "module" attribute to the script tag in modern browsers, the load is always deferred. This is a good thing, however it means our refresh-on-restart script must be savvy and wait for apos to pop up. It's the only piece of javascript that has any business being outside the bundle.